attest/core/src/error.rs

```diff
@@ -236,6 +237,22 @@ pub enum QuoteError {
     InvalidSize(u32),
     /// The base64 encoder did not output valid UTF-8 data
     InvalidUtf8,
+    /// Error converting to Quote3: {0}
```

Comment:
Personally I find the `Error` part redundant sine it's usually implied from the context in which this is printed, so I think `Quote3: {0}` is sufficient. Otherwise you end up with stuff like `Error while frobnicating: Error converting to Quote3: Error serializing: Expected blah blah blah` instead of `Frobnicating: Quote3: Serializing: Expected blah blah`

Reply:
Thanks!

Removed there and in the `QlError`